install_config/registry/deploy_registry_existing_clusters.adoc

```diff
@@ -91,7 +94,147 @@ Daemonsets ensure that when nodes are created, they contain copies of a
 specified pod. When the nodes are removed, the pods are garbage collected.
 
 For more information on `DaemonSets`, see
-xref:../../dev_guide/daemonsets.adoc#dev-guide-daemonsets[Using Daemonsets].
+xref:../../dev_guide/daemonsets.adoc#dev-guide-daemonsets[Using DaemonSets].
+
+[[redeploy-registry]]
+=== Re-deploying the Registry
+
+There is a couple of things to keep on mind when re-deploying an existing docker-registry
```

Comment:
s/is/are/

(docs team may have a better suggestion on cleaning up the informal language)